packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -384,6 +410,48 @@ export function ArticleMetadataPanel({
             />
           </Panel>
         )
+      case MetaDataType.Properties:
+        return (
+          <Panel>
+            <ControlLabel>{t('articleEditor.panels.properties')}</ControlLabel>
+            <ListInput
```

Comment:
I think we should also add a little info text here similar to the social media metadata. It should read something like "Properties can be set per article/page. Public properties are accessible through the public API"